overlord/snapstate/backend_test.go

```diff
@@ -397,18 +397,16 @@ func (f *fakeSnappyBackend) LinkSnap(info *snap.Info) error {
 	return nil
 }
 
-func (f *fakeSnappyBackend) StartSnapServices(info *snap.Info, meter progress.Meter) error {
+func (f *fakeSnappyBackend) StartServices(svcs []*snap.AppInfo, meter progress.Meter) error {
 	f.ops = append(f.ops, fakeOp{
-		op:   "start-snap-services",
-		name: info.MountDir(),
+		op: "start-snap-services",
```

Comment:
this is a bit "cheating", if one now replaces all the .Services() with nil tests would still pass. We probably want a couple of tests with services in snapstate now and then you can have the ifs, notice that the way to achieve that is to have a name like service-snap handled in backend_test.go in fake ReadInfo